_data/domains.yml

```diff
@@ -284,7 +284,7 @@ websites:
       software: Yes
       doc: https://blogg.loopia.se/2015/03/16/nu-kan-du-logga-in-med-tvafaktorsautentiering-hos-loopia-genom-bankid/
       exceptions:
-          text: "TFA only available for Swedish citizens"
+          text: "TFA only available for Swedish residents with BankID."
```

Comment:
So you don't have to be a Swedish citizen to use BankID? Because it asks for your 'personnummer' which only works with Swedish citizens